project/profiles/capacity4more/modules/c4m/ds/c4m_ds/c4m_ds.pages.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ */
+
+/**
+ * Form definition for admins to manage the disclaimer text.
+ */
+function c4m_ds_manage_disclaimer_form($form, $form_state) {
+  $form['c4m_ds_content_disclaimer'] = array(
+    '#title' => t('Disclaimer text'),
+    '#type' => 'textarea',
+    '#default_value' => variable_get('c4m_ds_content_disclaimer', 'DISCLAIMER: This information is provided in the interests of knowledge sharing and capacity development and should not be interpreted as the official view of the European Commission, or any other organisation.'),
```

Comment:
@soniCaH 
On the ticket it is written explicitly to use this variable name
![selection_273](https://cloud.githubusercontent.com/assets/5985683/18057106/4f3f4ed8-6e17-11e6-9508-bb3a49f74a79.png)